src/gui/guiButtonKey.cpp

```diff
@@ -0,0 +1,93 @@
+#include "guiButtonKey.h"
```

Comment:
Please add the SPDX header. Also feel free to add your own name as copyright holder.

Reply:
[f0cd316](https://github.com/luanti-org/luanti/pull/15791/commits/f0cd316266e97658e7dbb96741259f560d502db8)